src/OpenLoco/Windows/PromptBrowseWindow.cpp

```diff
@@ -899,7 +899,7 @@ namespace OpenLoco::Ui::PromptBrowse
             }
 
             // Create full path to target file.
-            fs::path path = fs::path(&_directory[0]) / inputSession.buffer;
+            fs::path path = fs::u8path((char*)_directory) / inputSession.buffer;
```

Comment:
This removes the explicit unpacking of the `loco_global` with a C-style cast. Any particular reason for this? I'm not sure this is better.